doc/api_methods.md

```diff
@@ -70,14 +70,14 @@ Returns an array of layers in the map. The first is the base layer.
 
 ### vis.addOverlay(_options_)
 
-Adds an overlay to the map that can be either a zoom control, a tooltip or an infobox.
+Adds an overlay to the map that can be either a zoom control, a tooltip or an infowindow (hover action).
 
```

Comment:
Here's how an infobox looks like:

<img width="506" alt="screen shot 2016-02-01 at 16 53 20" src="https://cloud.githubusercontent.com/assets/390398/12722262/61ae00c2-c904-11e5-80c8-23d7237771f6.png">

It basically appears when you hover a feature and displays some info about that featured, but it's always displayed in the same position (bottom left of the window in :point_up: case).